subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/DefaultVersionedComponentChooserTest.groovy

```diff
@@ -232,6 +233,7 @@ class DefaultVersionedComponentChooserTest extends Specification {
 
     }
 
+    @Ignore(/* Fixme @melix */)
```

Comment:
@melix, need your help here

Reply:
Fixed, the problem was the `with` clause, for some reason its param was `null`.
Luckily it has a `this` as well, used that instead and extracted to a separate methods for clarity.